API/chat/tests.py

```diff
@@ -670,13 +617,9 @@ def test_delete_message_that_does_not_exist(self):
             timestamp=timestamp
         )
 
-        qstring = urllib.urlencode({
-            'id': message.id + 1
-        })
-
         response = self.client.delete(
-            reverse('chat:message', args=(self.channel.name,)),
-            qstring
+            reverse('chat:message', args=(message.id + 1,)),
+            {}
```

Comment:
You have 2 different urls with different number of parameters that share the same name `"message"`. Is that intentional? I think it's a good idea to use different names and I think it will make this problem go away.